accname/name/comp_host_language_label.tentative.html

```diff
@@ -0,0 +1,118 @@
+    <h3>HTML input label/for</h3>
+    <label for="irn">input label</label>
+    <input id="irn" type="text" data-expectedlabel="input label" data-testname="html: label[for]input[role=none]" class="ex-label" role="none">
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: input[role=none] with label encapsulation" class="ex-label" role="none">
+    </label>
+
+    <h2>HTML input with role="none" and label with role none (presentational roles conflict resolution)</h2>
+    <h3>HTML input label/for</h3>
+    <label role="none" for="lirp">input label</label>
+    <input id="lirp" type="text" data-expectedlabel="input label" data-testname="html: label[for][role=none] input[role=none]" class="ex-label" role="none">
+    <h3>HTML label encapsulation</h3>
+    <label role="none">
+      <span>input label</span>
+      <input type="text" data-expectedlabel="input label" data-testname="html: input[role=none] with label[role=none] encapsulation" class="ex-label" role="none">
+    </label>
+
+    <!-- This test case is specifically designed to check the accessible name, but a separate test case has been created in wai-aria/role/role_none_conflict_resolution.tentative.html to check the role -->
+    <h2>HTML input with role="none" and disabled with label associated</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irpd">input label</label>
+    <input id="irpd" type="text" data-expectedlabel="" data-testname="html: label[for]input[role=none][disabled]" class="ex-label" role="none" disabled>
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="" data-testname="html: input[role=none][disabled] with label encapsulation" class="ex-label" role="none" disabled>
+    </label>
+
+    <h2>HTML img with role="none" and non empty alt</h2>
+    <img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" role="none" alt="w3c logo" data-expectedlabel="" data-testname="html: img[role=none][non-empty alt]" class="ex-label">
+
+    <h2>SVG wrapping title with role="none"</h2>
+    <svg viewBox="0 0 20 10" xmlns="http://www.w3.org/2000/svg">
+      <circle cx="5" cy="5" r="4" data-expectedlabel="circle" data-testname="svg: svg circle title[role=none]" class="ex-label">
+        <title role="none">circle</title>
+      </circle>
+    </svg>
+
+    <h2>HTML fieldset wrapping legend with role="none"</h2>
+    <fieldset data-expectedlabel="legend" data-testname="html: fieldset wrapping legend[role=none]" class="ex-label"><legend role="none">legend</legend></fieldset>
+
+    <h2>HTML fieldset with role="none" wrapping legend</h2>
+    <fieldset role="none" data-expectedlabel="" data-testname="html: fieldset[role=none] wrapping legend" class="ex-label"><legend>legend</legend></fieldset>
```

Comment:
I don't think this is a valid test, as if the role is none, there is no mandate in WebDriver to have a backing object or return a label. I would remove this test and possible refile as a new "blocked by testability" issue.